pkg/util/names/names.go

```diff
@@ -11,21 +11,22 @@ import (
 )
 
 const (
-	// KubernetesReservedNSPrefix is the prefix of namespace which reserved by Kubernetes system, such as:
-	// - kube-system
-	// - kube-public
-	// - kube-node-lease
-	KubernetesReservedNSPrefix = "kube-"
```

Comment:
I'm afraid we can't list all Kubernetes reserved namespaces, so I suggest keeping this prefix and all namespaces with this prefix should be ignored by default.

Reply:
ok，you are right!
The k8s docs is said that `Avoid creating namespaces with the prefix kube-, since it is reserved for Kubernetes system namespaces.`[namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#working-with-namespaces)
I will fix it